stdlib/ballerina-grpc/src/main/java/org/ballerinalang/net/grpc/proto/ServiceProtoUtils.java

```diff
@@ -149,7 +154,7 @@ private static Service getUnaryServiceDefinition(ServiceNode serviceNode, File.B
         Service.Builder serviceBuilder = Service.newBuilder(serviceNode.getName().getValue());
         
         for (ResourceNode resourceNode : serviceNode.getResources()) {
-            Message requestMessage = getRequestMessage(resourceNode);
+            Message requestMessage = getRequestMessage(resourceNode,fileBuilder);
```

Comment:
Do we need to pass fileBuilder. we need to handle nested message types in request msg to handle enum without passing fileBuilder